services/apigateway/src/it/java/software/amazon/awssdk/services/apigateway/GetExportIntegrationTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.apigateway;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.testutils.Waiter;
+
+/**
+ * Integration tests for {@link software.amazon.awssdk.services.apigateway.model.GetExportRequest}.
+ */
+@Ignore // Running during release causing TooManyRequests exception
```

Comment:
Is there any way we can remove `ignore`?

Reply:
Going to reach out to APIG today to get our limits increased so we can remove this.